docs/source/markdown/podman-image-mount.1.md

```diff
@@ -28,26 +33,31 @@ Mount all images.
 
 #### **--format**=*format*
 
-Print the mounted images in specified format (json).
+Print the mounted images in specified format (`JSON`).
```

Comment:
`JSON` should also be small.

Reply:
From what I can see caps is correct.... https://www.rfc-editor.org/info/std90

Reply:
```
$ podman image diff --format JSON alpine
Error: only supported value for '--format' is 'json'
$ podman image diff --format json alpine
<works>
````

Writing it in caps lock does not work as you can see. I think it has to be defined what is more important, that it is written in caps like an acronym is written or that the right input for the flag is listed. I think that the later variant is more important. You can still describe it in the description of the option as `JSON` but if you list the argument for the flag in a table or refer to it should be small.

For the description should also be mentioned what happens if you don't right input after the equal sign. JSON is the only xml like format that can be selected here, but if you just leave is blank the output is plain text.

Btw. I think that the flag does not even work. Input after the equal sign is simply ignored.

Reply:
I agree, there is something buggy about this. And the examples use `diff `when they should be using `image diff`.  I remember testing the command on my machine and thought I was doing something wrong. I am going test this command more and open an issue

Reply:
Only the lowercase version is accepted so we should not show this as `JSON`